pages/concepts/webhook-events.mdx

```diff
@@ -86,84 +86,3 @@ Webhooks are available for the list of Events shown below. As API enhancements a
   below. Refer to the <a href="https://docs.dwolla.com/#events">API docs</a> for
```

Comment:
We would need to update the alert text in this `<AlertBar>`.

Reply:
ty!